src/main/kotlin/com/ioki/lokalise/gradle/plugin/internal/LokaliseApi.kt

```diff
@@ -9,14 +9,23 @@ import kotlinx.coroutines.coroutineScope
 import kotlinx.coroutines.delay
 import org.gradle.api.GradleException
 
-internal class LokaliseApi(
+interface LokaliseApi {
```

Comment:
💭 Since the `LokaliseApi` is a wrapper around the `Lokalise` object from our dependency, we might want to add `downloadFiles` to this too..
Then we can also use this class in the `DownloadTask` then..

What do you think? 🤔 

I think it is out of scope of this PR, but still worth to think about...

Reply:
Taking a brief look at the `DownloadTask` it seems like adding that abstraction might help in making the task a bit structured as a few parts would not be needed in the task itself anymore.

I will open an issue to continue discussion there.